indico/modules/events/papers/templates/display/_paper_list.html

```diff
@@ -0,0 +1,68 @@
+{% from 'events/management/_lists.html' import render_displayed_entries_fragment %}
+
+{% macro render_user_papers_list(event, contributions, to_review=false, reviewed=false) %}
+    <section>
+    {% if to_review %}
+        {% set reviewing_list_id='to-review-list' %}
+    {% elif reviewed %}
+        {% set reviewing_list_id='reviewed-list' %}
+    {% else %}
+        {% set reviewing_list_id='' %}
+    {% endif %}
+        <div id="{{ reviewing_list_id }}" class="display-contribution-list">
+            <ul>
+                {% for contribution in contributions %}
+                    {% set latest_revision = contribution._paper_last_revision %}
```

Comment:
Would probably be cleaner to access it through the proxy class instead of using the underscored one